include/tests_ssh

```diff
@@ -97,6 +97,9 @@
                 IgnoreRhosts:YES,,NO\
                 UseDNS:YES,,NO\
                 X11Forwarding:NO,,YES\
+                FingerprintHash:SHA256,MD5,\
+                TCPKeepAlive:YES,,NO\
+                Compression:DELAYED,YES,NO\
```

Comment:
I was thinking of this. And on one hand have performance and on the other, vunerabillities.
I don't know them to.

How about chanigng it in reverse? I mean `Compression no` would be recomended, `delayed` would be medium scored and `yes` -- wrong one?